integration_tests/fixtures.py

```diff
@@ -31,21 +29,7 @@
 logger = prestodb.logging.get_logger(__name__)
 
 
-def get_latest_release():
-    resp = requests.get(
-        "https://repo1.maven.org/maven2/com/"
-        "facebook/presto/presto-server/maven-metadata.xml"
-    )
-    xml_root = ET.fromstring(resp.content)
-    try:
-        return xml_root.find("versioning").find("release").text.strip()
-    except AttributeError:
-        raise Exception("not release found")
-
-
-# Temporary fix because can't start 0.203
-# Use get_latest_release() instead
-PRESTO_VERSION = os.environ.get("PRESTO_VERSION") or get_latest_release()
+PRESTO_VERSION = os.environ.get("PRESTO_VERSION") or "latest"
```

Comment:
"latest" is not a presto version, so

```suggestion
PRESTO_IMAGE_VERSION = os.environ.get("PRESTO_VERSION") or "latest"
```

Reply:
let's actually have
```
PRESTO_VERSION = os.environ.get("PRESTO_VERSION") or "329"
```

here